.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -149,20 +149,26 @@ commands:
               fi
 
               # new relic automation
-              if [ "${nrURL}" != "" ]; then
-                nrURL="--url ${nrURL}"
-              else
+              if [ "${nrURL}" == "" ]; then
                 if [ "${actionName}" == "" ]; then
+                  # extract action name from package name
                   actionName=`node -e "console.log(require('./package.json').name.replace('@adobe/helix-', ''))"`
                 fi
+                # use package version as action version
                 actionVersion=`node -e "console.log(require('./package.json').version.match(/^[0-9]+/)[0])"`
-                nrURL="--url https://adobeioruntime.net/api/v1/web/${actionNS}/${actionPackage}/${actionName}@v${actionVersion}/_status_check/healthcheck.json"
+                # assemble runtime url
+                nrURL="https://adobeioruntime.net/api/v1/web/${actionNS}/${actionPackage}/${actionName}@v${actionVersion}/_status_check/healthcheck.json"
+              fi
+              if [ "${nrName}" == "" ]; then
+                # use package name
+                nrName=`node -e "console.log(require('./package.json').name)"`
               fi
               if [ $aws = true ]; then
```

Comment:
This line will only work if `aws` variable has been set, otherwise the expression becomes `if [ = true ]; then` which will cause an error. In order to avoid such errors, it has become customary to always enclose variables in conditionals with quotes, so better use: `if [ "$aws" = true ]; then`

Reply:
Good to know, thanks. But since we always define `aws` as either `true` or `false` above, this should be safe, at least for now.